src/ranged.cpp

```diff
@@ -341,29 +424,52 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
     return attack;
 }
 
+static double erfinv( double x )
+{
+    const double epsilon = 1e-6;
+    double z = 0.0;
+
+    // find erf(z) - x = 0
+    // d/dz ( erf(z) - x ) = 2/sqrt(pi) . e^(-z^2)
+
+    for( int n = 0; n < 10; ++n ) {
```

Comment:
Is this the change that leads to the smoother graph in 18943? Why does it need to be iterative?